src/Services/API/TaskTracking/TaskTrackingService.Contract/DataTransfer/TaskCreateDto.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace TaskTrackingService.Contract.DataTransfer;
+    public class TaskCreateDto
+    {
+    public string Title { get; set; } = null!;
+    public string Description { get; set; } = string.Empty;
+    public long  UserId { get; set; }
+    public string EnterpriseId { get; set; } = null!;
```

Comment:
use enterpise id from auth context (User.GetAuthContext().EnterpriseId) instead of enterprise id from user input